scripts/integration-tests/test-curiosity.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eu
+
+if [[ -z "${CURIOSITY_ENDPOINT:-}" ]]; then
+    scriptdir=$(realpath -s "$0" | xargs dirname)
+    echo "Missing curiosity endpoint."
+    echo "Did you read $scriptdir/readme.md ?"
+    exit 1
+fi
+
+echo "[+] Querying curiosity root endpoint"
```

Comment:
I forgot to ask last time: is the `[+]` syntax some specific convention for hivemind ?

Reply:
That's a very good question. It's a de-facto convention I copied without ever questioning. I couldn't find its origin with a quick google search.

The "best" response I found is this one: https://stackoverflow.com/a/21655625

I'd be interesting to know where this convention originates from.